js/google_ima.js

```diff
@@ -2387,6 +2389,35 @@ require("../html5-common/js/utils/utils.js");
     };
 
     /**
+     * Checks to see if autoplay requires the video to be muted
+     * @public
+     * @method TemplateVideoWrapper#requiresMutedAutoplay
+     * @param {boolean} true if video must be muted to autoplay, false otherwise
+     */
+    this.requiresMutedAutoplay = function() {
+      return OO.isSafari || OO.isIos || OO.isAndroid;
```

Comment:
Do we want to target only Safari 11 here, or just be consistent across versions?

Reply:
I'll add that in.